flooder/src/main.rs

```diff
@@ -104,48 +145,47 @@ fn main() -> Result<(), anyhow::Error> {
     Ok(())
 }
 
-fn flood(
-    pool: &[Api<sr25519::Pair, WsRpcClient>],
+fn flood<'a>(
+    pool: impl Fn(usize) -> &'a Api<sr25519::Pair, WsRpcClient> + Sync,
     txs: impl IndexedParallelIterator<Item = TransferTransaction>,
```

Comment:
Not sure how it slipped but since this is ParallelIterator the rate_limiting does not do what we want it to do. It should be just a vector or normal iterator.

Reply:
Please see the new version.